src/compack.f

```diff
@@ -1,60 +1,33 @@
+!>    @file
+!>    @brief This subroutine packs up a data field using a complex
+!>    packing algorithm as defined in the GRIB2 documention.
+!>    @author Gilbert ORG: W/NP11 @date 2000-06-21
+!>     
+!>    It supports GRIB2 complex packing templates with or without
```

Comment:
OK, the information in your brief tag needs to be in two places, for the file and for the subroutine. Look at some of the files I did for examples of how to do this.